tests/ui/issues/issue-34209.stderr

```diff
@@ -5,7 +5,12 @@ LL | enum S {
    | ------ variant `B` not found here
 ...
 LL |         S::B {} => {},
-   |            ^ help: there is a variant with a similar name: `A`
+   |            ^
+   |
+help: there is a variant with a similar name
+   |
+LL |         S::A {} => {},
+   |            ~
```

Comment:
Tangent unrelated to this PR: I do wonder if the similarity detector should not kick in here, for a single-letter identifier. `A` and `B` are not that similar, they're 100% different!

Reply:
Yeah, I think that we do limit the levenshtein distance logic to N>3, but we still want to mention "hey, you might have meant to use another available variant". For that maybe we should just list all variants, maybe?